consensus/polybft/extra.go

```diff
@@ -0,0 +1,390 @@
+package polybft
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/go-hclog"
+
+	bls "github.com/0xPolygon/polygon-edge/consensus/polybft/signer"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/bitmap"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/fastrlp"
+)
+
+const (
+	// ExtraVanity represents a fixed number of extra-data bytes reserved for proposer vanity
+	ExtraVanity = 32
+
+	// ExtraSeal represents the fixed number of extra-data bytes reserved for proposer seal
+	ExtraSeal = 65
+)
+
+var (
+	// PolyMixDigest represents a hash of "PolyBFT Mix" to identify whether the block is from PolyBFT consensus engine
+	PolyMixDigest = types.Hash(ethgo.HexToHash("adce6e5230abe012342a44e4e9b6d05997d6f015387ae0e59be924afc7ec70c1"))
```

Comment:
There is no need to use `ethgo` for hex to hash, we have helper methods in the `types` module that do this exact same thing already

Reply:
Agree.